### PR TITLE
Make edits to the demo data section 

### PR DESCRIPTION
**Pull request by mluu12**
Closes https://github.com/influxdata/docs-v2/issues/1385

Moved demo data section to its own file, and organized the write data index. 

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/get-started.md [hunk 1]
```diff
@@ -13,19 +13,39 @@ influxdb/cloud/tags: [get-started, install]
...
+
+To write data into InfluxDB, you need the following:
+ 
+- **organization** – _See [View organizations](/influxdb/cloud/organizations/view-orgs/#view-your-organization-id)
+ for instructions on viewing your organization ID._
+- **bucket** – _See [View buckets](/influxdb/cloud/organizations/buckets/view-buckets/) for
+ instructions on viewing your bucket ID._
+- **API token** – _See [View tokens](/influxdb/cloud/security/tokens/view-tokens/)
+ for instructions on viewing your API token._
+- **InfluxDB Cloud region URL** – _See [InfluxDB Cloud regions](/influxdb/cloud/reference/regions/)_.
```
Comment (kelseiv) [inline content/influxdb/cloud/get-started.md:26]:
```suggestion
```

File: content/influxdb/cloud/write-data/developer-tools/line-protocol.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  influxdb_cloud:
+    name: Write with line protocol
+    parent: Developer tools
+weight: 204
+related:
+  - /influxdb/cloud/reference/syntax/line-protocol/
+  - /influxdb/cloud/reference/cli/influx/write/
+---
+
+Use line protocol to write data to InfluxDB Cloud. Line protocol is a is a text-based format that provides the measurement, tag set, field set, and timestamp of a data point. Learn more about [line protocol](/influxdb/v2.0/reference/syntax/line-protocol/).
```
Comment (kelseiv) [inline content/influxdb/cloud/write-data/developer-tools/line-protocol.md:15]:
```suggestion
Use line protocol to write data to InfluxDB Cloud. Line protocol is a text-based format that provides the measurement, tag set, field set, and timestamp of a data point. Learn more about [line protocol](/influxdb/v2.0/reference/syntax/line-protocol/). 
```

File: content/influxdb/cloud/write-data/developer-tools/line-protocol.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+1. Verify the line protocol adheres to the following conventions:  
+   - Each line represents a data point.
+   - Each data point requires a:  
+     - [*measurement*](/influxdb/cloud/reference/syntax/line-protocol/#measurement)
+     - [*field set*](/influxdb/cloud/reference/syntax/line-protocol/#field-set)
+     - (Optional) [*tag set*](/influxdb/cloud/reference/syntax/line-protocol/#tag-set) 
+     - [*timestamp*](/influxdb/cloud/reference/syntax/line-protocol/#timestamp) 
+2. Click **Data** in your left navigation bar. 
+3. Select **Line Protocol** and choose your bucket. 
+4. Select your **Precision** in the dropdown menu. By default, the precision is set to nanosections.
```
Comment (kelseiv) [inline content/influxdb/cloud/write-data/developer-tools/line-protocol.md:34]:
```suggestion
4. Select your **Precision** in the dropdown menu. By default, the precision is set to nanoseconds. 
```

File: content/influxdb/cloud/write-data/developer-tools/line-protocol.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+     - (Optional) [*tag set*](/influxdb/cloud/reference/syntax/line-protocol/#tag-set) 
+     - [*timestamp*](/influxdb/cloud/reference/syntax/line-protocol/#timestamp) 
+2. Click **Data** in your left navigation bar. 
+3. Select **Line Protocol** and choose your bucket. 
+4. Select your **Precision** in the dropdown menu. By default, the precision is set to nanosections. 
+5. Do one of the following: 
+   - Upload your line protocol file. 
+   - Write your line protocol manually, and then click **Submit**. 
+
+## Influx write command
```
Comment (kelseiv) [inline content/influxdb/cloud/write-data/developer-tools/line-protocol.md:39]:
```suggestion
## Write line protocol using the influx write command
```

File: content/influxdb/cloud/write-data/developer-tools/line-protocol.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+
+Use the [`influx write` command](/influxdb/v2.0/reference/cli/influx/write/) to write data using line protocol
+to InfluxDB. 
+Set the path for your `--file` flag and set the path for your `--format`. 
+Verify that your line protocol follows the correct conventions. 
+
+#### Example write command
+```sh
+influx write -b example-bucket -f path/to/example.csv
```
Comment (kelseiv) [inline content/influxdb/cloud/write-data/developer-tools/line-protocol.md:48]:
```suggestion
influx write -b example-bucket -f path/to/example.txt 
```

File: content/influxdb/cloud/write-data/developer-tools/line-protocol.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+4. Select your **Precision** in the dropdown menu. By default, the precision is set to nanosections. 
+5. Do one of the following: 
+   - Upload your line protocol file. 
+   - Write your line protocol manually, and then click **Submit**. 
+
+## Influx write command
+
+Use the [`influx write` command](/influxdb/v2.0/reference/cli/influx/write/) to write data using line protocol
+to InfluxDB. 
+Set the path for your `--file` flag and set the path for your `--format`.
```
Comment (kelseiv) [inline content/influxdb/cloud/write-data/developer-tools/line-protocol.md:43]:
```suggestion
Set the path to the line protocol file using the `-f` flag.
```

File: content/influxdb/cloud/write-data/developer-tools/line-protocol.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+5. Do one of the following: 
+   - Upload your line protocol file. 
+   - Write your line protocol manually, and then click **Submit**. 
+
+## Influx write command
+
+Use the [`influx write` command](/influxdb/v2.0/reference/cli/influx/write/) to write data using line protocol
+to InfluxDB. 
+Set the path for your `--file` flag and set the path for your `--format`. 
+Verify that your line protocol follows the correct conventions.
```
Comment (kelseiv) [inline content/influxdb/cloud/write-data/developer-tools/line-protocol.md:44]:
```suggestion
Verify that your line protocol follows the correct [line protocol format](#line-protocol-format) conventions. 
```

File: content/influxdb/cloud/get-started.md [hunk 8]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
 
-After you've [signed up for InfluxDB Cloud](/influxdb/cloud/sign-up/), you're ready to start writing, querying, and visualizing data.
-
-To get started working in Cloud, we recommend using demo data and notebooks.
+After you've [signed up for InfluxDB Cloud](/influxdb/cloud/sign-up/), you're ready to get started:
+
+1. Do one of the following:
+   - Add [demo data](#add-demo-data).
+   - Add [sample data](#add-sample-data). 
+   - Use your own data to explore Cloud. See how to [write your data](/influxdb/cloud/write-data/) to Cloud.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:19]:
```suggestion
   - [Use your own data](/influxdb/cloud/write-data/) to explore InfluxDB Cloud.
```

File: content/influxdb/cloud/get-started.md [hunk 9]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
+2. [Create a notebook](#create-a-notebook) by clicking **Books > Create Notebook**, and then do the following: 
+     1. [Select metrics](#select-metrics)
+     2. [Visualize data](#visualize-data)
+     3. (Optional) [Process data](#process-data)
+     4. (Optional) [Monitor data](#monitor-data)
+     5. (Optional) [Output to a new bucket and export as a task](#output-to-a-new-bucket-and-export-as-a-task)
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:25]:
```suggestion
2. [Create a notebook](#create-a-notebook):
    1.  Clicking **Books > Create Notebook** in the navigation menu on the left.
        {{< nav-icon "books" >}}
    2. [Select metrics](#select-metrics)
    3. [Visualize data](#visualize-data)
    4. (Optional) [Process data](#process-data)
    5. (Optional) [Monitor data](#monitor-data)
    6. (Optional) [Output to a new bucket and export as a task](#output-to-a-new-bucket-and-export-as-a-task)
```

File: content/influxdb/cloud/get-started.md [hunk 10]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
 
 2.  Click **{{< icon "plus" >}} Add Demo Data**, and then select the **Website Monitoring** bucket.
 3.  The Demo Data bucket appears in your list of buckets.
 
-## Select metrics
+## Add sample data
 
-Now that we've added the demo data bucket, we're ready to create our notebook.
+{{% note %}}
+For example purposes, we will be referencing the [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data), but we provide other [sample data sets](/influxdb/cloud/reference/sample-data/#sample-datasets) as well.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:41]:
```suggestion
The examples below use the [NOAA NDBC sample data](/influxdb/v2.0/reference/sample-data/#noaa-ndbc-data), but we provide other [sample data sets](/influxdb/cloud/reference/sample-data/#sample-datasets) as well.
```

File: content/influxdb/cloud/get-started.md [hunk 11]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
+   - [Install community templates](#add-sample-data-with-community-templates) 
+   - [Download using the InfluxData UI](#add-sample-data-with-influxdata-ui)
+
+### Add sample data with community templates 
+
+After installing the template, sample data will be stored in the sample data bucket. 
+
+1. Go to the community templates respository on GitHub and head to our [sample data documentation](/influxdata/community-templates/tree/master/sample-data). 
+2. [Install the template](https://github.com/influxdata/community-templates/tree/master/sample-data#quick-install) with the instructions given. 
+
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:57]:
```suggestion
### Add sample data with community templates 

1. Visit the **InfluxDB templates page** in the InfluxDB Cloud UI. Click **Settings** > **Templates** in the navigation menu on the left.

    {{< nav-icon "settings" >}}
      
2. Paste the Sample Data community temple URL in **resource manifest file** field:

    ```
    https://github.com/influxdata/community-templates/blob/master/sample-data/sample-data.yml
    ```

The sample data template installs a task that collects sample data and dashboards to visualize the sample data.
After installing the template, sample data is downloaded and stored in the sample data bucket at regular intervals.
```

File: content/influxdb/cloud/get-started.md [hunk 12]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
+### Add sample data in InfluxDB Cloud UI
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:58]:
```suggestion
### Write sample data with an InfluxDB task
```

File: content/influxdb/cloud/get-started.md [hunk 13]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
 
-    {{< nav-icon "notebooks" >}}
-2. Click **+Create Notebook**.
-3. Enter a name for your notebook in the **Name this notebook** field.
-4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](#cell-types) below for details.)
-5. In the **Metric Selector** cell, click the **Select a bucket** dropdown and choose **Website Monitoring Bucket**.
+1. Click **Data > Bucket** and click **Create Bucket** to create a bucket and name it **noaa**.
+2. Do one of the following to download sample data: 
+   - [Install community templates](#add-sample-data-with-community-templates) 
+   - [Download using the InfluxData UI](#add-sample-data-with-influxdata-ui)
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:49]:
```suggestion
  - [Write sample data with an InfluxDB task](#write-sample-data-with-an-influxdb-task)
```

File: content/influxdb/cloud/get-started.md [hunk 14]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
+    }
+
+    sample.data(set: "noaa")
+      |> to(
+          org: "example-org",
+          bucket: "example-bucket"
+      )
+    ```
+   Click here for more information about [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data), or other [sample data sets](/influxdb/v2.0/reference/sample-data/). 
+4. Click **Submit**.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:78]:
```suggestion
Use the [Flux InfluxDB sample package](/{{< latest "flux" >}}/stdlib/influxdata/influxdb/sample/) to download and write sample data to InfluxDB.
Add the following as an [InfluxDB task](/influxdb/cloud/process-data/manage-tasks/create-task/).

```js
import "influxdata/influxdb/sample"

option task = {
  name: "Collect NOAA NDBC data"
  every: 15m,
}

sample.data(set: "noaa")
  |> to(bucket: "noaa"  )
    ```
For more information about this and other InfluxDB sample datasets, see [InfluxDB sample data](/influxdb/cloud/reference/sample-data/). 
```

File: content/influxdb/cloud/get-started.md [hunk 15]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
 
-A common processing task is downsampling data to view it at different levels of granularity and reduce overall data disk usage over time. The following steps use a Flux script to downsample the website monitoring sample data to every 12 hours.
+A common processing task is downsampling data to view it at different levels of granularity and reduce overall data disk usage over time. The following steps use a Flux script to downsample your data to every 3 hours 30 seconds.
+
+For more information on how to process data, see [here](/influxdb/cloud/process-data/).
 
 1. Click **+** to add a cell, then select **Transform > Downsample**.
 2. In the **Apply aggregate** field, select **mean**.
-3. In the **Every Window Period** field, enter a value. For this example, we'll use `12h`.
+3. In the **Every Window Period** field, enter a value. For this example, we'll use `3h30s`, but any duration is supported by InfluxDB Cloud. For more information, see [duration literals](/influxdb/v2.0/reference/flux/language/lexical-elements/#duration-literals).
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:114]:
```suggestion
3. In the **Every Window Period** field, enter a value. For this example, we'll use `3h30s`, but any duration is supported by InfluxDB Cloud. For more information, see [duration literals](/{{< latest "flux" >}}/spec/lexical-elements/#duration-literals).
```

File: content/influxdb/cloud/get-started.md [hunk 16]
```diff
@@ -11,66 +11,130 @@ weight: 2
...
 4. Click **Preview** to view the downsampled data.
 
-## Output to a new bucket and export as a task
+### Monitor data
+
+Get notifications every time your data crosses a threshold. Specify your alerts by time and decide how the information will be sent to you. 
+
+1. Click **+** to add a cell, then select **Output > Alert**.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:121]:
```suggestion
1. In your notebook, click **{{< icon "plus" >}}** to add a cell, then select **Output > Alert**.
```

**Review by sanderson — CHANGES_REQUESTED** [review]

**Review by sanderson — APPROVED** [review]

**Review by kelseiv — APPROVED** [review]

**Review by kelseiv — APPROVED** [review]

**Pull request merged**